back-end/src/reservations/reservations.controller.js

```diff
@@ -101,16 +94,20 @@ function timeIsHourBeforeClosing(req, res, next) {
 }
 
 function dateAndTimeInFuture(req, res, next) {
-  const { data: { reservation_date } = {} } = req.body;
+  const {
+    data: {
+      reservation_date,
+      currentDate = "2024-04-18",
```

Comment:
this is probably removed in main but JIC want to make sure these are being passed down and not declared here